### ISSUE TITLE
PostgreSQL bytea returned as hex string

### ISSUE THREAD

**Issue by pgimeno** [keplerproject/luasql#84]
For example:

```sql
create table test(col bytea);
insert into test values ('a'::text::bytea);
select * from test;
```

shows

```
  x
------
 \x61
(1 rows)
```

When read using LuaSQL, the value returned is the same string displayed: `\x61` rather than the string `a`, even though Lua supports binary strings. This is highly inconvenient because it forces the application to decode the hexadecimal string, which has a high impact in performance (3x in my application).

I assume that a similar situation happens when writing data (which thankfully I don't need to use), but since it can only be done through a valid SQL statement, there's no way around that until prepared statements are available.

**Comment by tomasguisasola** [comment]
Yes, the driver to PostgreSQL was made with simplicity in mind.  There is no conversion of database types to Lua types.
